samples/Sentry.Samples.Maui/CtMvvmViewModel.cs

```diff
@@ -0,0 +1,15 @@
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+
+namespace Sentry.Samples.Maui;
+
+public partial class CtMvvmViewModel : ObservableObject
+{
+    // we auto-instrument how
```

Comment:
`how`? Is this left over from before the solution was implemented?

Maybe worth expanding on this for SDK users:
// Sentry automatically creates spans for code like XYZ in Mvvm apps